gulpfile.js

```diff
@@ -215,10 +217,36 @@ const test = gulp.series(testUnitTests, testWeb);
 const testInt = gulp.series(testDebugger);
 
 async function packageVsix() {
-    fs.emptyDirSync(packagedir);
-    return vsce.createVSIX({
+    if (isPreviewBuild) {
+        // Set Preview extension settings
+        await npm(['pkg', 'set', 'name=powerplatform-vscode-preview']);
+        await npm(['pkg', 'set', 'displayName="Power Platform Tools [PREVIEW]"']);
+        await npm(['pkg', 'set', 'description="Unsupported extension for testing Power Platform Tools"']);
+
+        gulp.src('README.md')
```

Comment:
see above comment: single readme.md; if isPreviewBuild, inject the warning text

Reply:
Switched to single readme file via text replacement, and changed warning text to H2